v0/destinations/awin/transform.js

```diff
@@ -0,0 +1,87 @@
+const {
+  BASE_URL,
+  DESTINATION
+} = require("./config");
+const {
+  defaultRequestConfig,
+  CustomError,
+  getSuccessRespEvents,
+  handleRtTfSingleEventError,
+  checkInvalidRtTfEvents
+} = require("../../util");
+
+const { getEndpoint } = require("./utils");
+
+const responseBuilder = (message, { Config }) => {
+  const { advertiserId, eventsToTrack } = Config;
+  let endpoint = BASE_URL.concat("&merchant=", advertiserId);
+  // if the event is present in eventsToTrack List
+  if (eventsToTrack.includes(message.event)) {
```

Comment:
We should check if `eventsToTrack` is an array or not.

Reply:
Created a new Array and used it

Reply:
What if `eventsToTrack` is not an array?

Reply:
`eventsToTrack` is a dynamicCustomForm, which is in the format of an Array of Objects[ {...}, {...}, ...].
Anyway, I can add that validation.